core/services/repo_manager.rb

```diff
@@ -83,6 +87,9 @@ def attribute_name(product)
 
   def find_repository(product_name, product, box)
     @ui.info('Looking for repo')
+    @ui.warning('MDBCI cannot determine the existence/correctness of the specified version of the product!')
```

Comment:
This notification will be fired for both situations: when we CAN find the version and when we can not.